engine/api/workflow_hook.go

```diff
@@ -23,10 +26,68 @@ func (api *API) getWorkflowHooksHandler() Handler {
 
 func (api *API) getWorkflowHookModelsHandler() Handler {
 	return func(ctx context.Context, w http.ResponseWriter, r *http.Request) error {
+		vars := mux.Vars(r)
+		key := vars["key"]
+		workflowName := vars["permWorkflowName"]
+		nodeID, errN := requestVarInt(r, "nodeID")
+		if errN != nil {
+			return sdk.WrapError(errN, "getWorkflowHookModelsHandler")
+		}
+
+		p, errP := project.Load(api.mustDB(), api.Cache, key, getUser(ctx))
+		if errP != nil {
+			return sdk.WrapError(errP, "getWorkflowHookModelsHandler")
+		}
+
+		wf, errW := workflow.Load(api.mustDB(), api.Cache, key, workflowName, getUser(ctx))
+		if errW != nil {
+			return sdk.WrapError(errW, "getWorkflowHookModelsHandler")
+		}
+
+		node := wf.GetNode(nodeID)
+		if node == nil {
+			return sdk.WrapError(sdk.ErrWorkflowNodeNotFound, "getWorkflowHookModelsHandler")
+		}
+
 		m, err := workflow.LoadHookModels(api.mustDB())
 		if err != nil {
 			return sdk.WrapError(err, "getWorkflowHookModelsHandler")
```

Comment:
getWorkflowHookModelsHandler> Unable to load hook models

Reply:
workflow.LoadHookModels already get this information